tools/system_libs.py

```diff
@@ -2390,6 +2393,10 @@ def add_sanitizer_libs():
     add_library('libwasmfs')
 
   add_sanitizer_libs()
+
+  # Add forced stdlibs at the end so that the order of system libraries is
+  # still preserved above, even when EMCC_FORCE_STDLIBS=1 is used.
```

Comment:
Could this comment go on line 2295? It seems to apply to the other calls to this function as well. Though, the call on 2328 is not actually last - could it be?